src/server/models/flight.js

```diff
@@ -26,4 +31,12 @@ FlightSchema.statics = {
   }
 };
 
-export default mongoose.model('Flight', FlightSchema);
+FlightSchema.statics.getFlightFromFlightName = (flightname) => {
+  const flightNameArray = flightname.split('-');
```

Comment:
You could use RegExp or even native Node QueryString parser to do this, which would make it easier to handle garbage input. Right now, if your input doesn't contain the delimiter, you are going to be in trouble. Is it also possible to provide regexp validation on the model schema? (It's been a little while since I have used Mongoose, so I don't know off the top of my head.)